docs/general/builtins/bit-operators.rst

```diff
@@ -0,0 +1,40 @@
+.. highlight:: psql
+
+.. _bit-operators:
+
+=============
+Bit operators
+=============
+
+Bit :ref:`operators <gloss-operator>` perform bitwise operations on numeric
+integral values and :ref:`bit <data-type-bit>` strings:
+
+===========  ========================
+Operator     Description
+===========  ========================
+``&``        Bitwise AND of operands.
+``|``        Bitwise OR of operands.
+``^``/``#``  Bitwise XOR of operands.
+===========  ========================
+
+Here's an example that uses all of the available bit operators::
+
+    cr> select 1 & 2 | 3 AS n;
+    +-----+
+    |   n |
+    +-----+
+    |   3 |
+    +-----+
+    SELECT 1 row in set (... sec)
+
+When applied to numeric operands, bit operators always return the data type
+of the argument with the higher precision.
+
+If at least one operand is ``NULL``, bit operators return ``NULL``.
+
+When applied to ``BIT`` strings, operands must have equal length.
+
+.. NOTE::
+
+    Bit operators have the same precedence and evaluated from left to right.
```

Comment:
Kinda obvious but in PG [docs](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-PRECEDENCE) they have smth like 
> Add parentheses if you want an expression with multiple operators to be parsed in some other way than what the precedence rules imply.

So maybe extra clarification wouldn't hurt